package.json

```diff
@@ -133,6 +145,26 @@
       "import": "./handlers/next/index.mjs",
       "require": "./handlers/next/index.js"
     },
+    "./handlers/fetch-api": {
+      "types": "./handlers/fetch-api/index.d.ts",
+      "import": "./handlers/fetch-api/index.mjs",
+      "require": "./handlers/fetch-api/index.js"
+    },
+    "./handlers/next-app-router": {
+      "types": "./handlers/fetch-api/index.d.ts",
+      "import": "./handlers/fetch-api/index.mjs",
+      "require": "./handlers/fetch-api/index.js"
+    },
```

Comment:
won't it have the same names as fetch api? It would be confusing, probably better to also add alias on export

so `next-app-router` file can include `export { FetchApi as  NextAppRouterWebhook } from '../fetch-api'` or something like this

Reply:
would be easier if we see docs at this point

Reply:
This is fine, since exported names do not reference Fetch API in any way, names are the same as in Next.js handlers except they are in a different folder. Our exports are:
- `createAppRegisterHandler`
- `createManifestHandler`
- `createProtectedHandler`
- `SaleorSyncWebhook`
- `SaleorAsyncWebhook`